buildout.cfg

```diff
@@ -15,6 +15,7 @@ recipe = zc.recipe.egg
 eggs =
     aioconsole
     guillotina
+    cookiecutter
```

Comment:
not sure, if setup.py should depend on it because setup.py also installs the console_scripts...